clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -615,7 +615,7 @@ DPCT_ENUM_OPTION(
                                int(ExplicitNamespace::EN_SYCLCompat),
                                "Generate code with syclcompat:: namespace.",
                                false)),
-    llvm::cl::desc("Define the namespaces to use explicitly in generated "
+    llvm::cl::desc("Defines the namespaces to use explicitly in generated "
```

Comment:
let's use Define?  (keep same style)

Reply:
It has been updated in https://github.com/oneapi-src/SYCLomatic/pull/2341/files#diff-f694ef2a3d5a06cb1dcf9782cbebd18140f6218dfa6bcf59d7e06ec3a293e90fR618